oauth.rst

```diff
@@ -1,131 +1,157 @@
-Partner Integrations: Public Documentation
-=========================================
+DOCK OAuth Integration Guide
+============================
 
-The OAuth 2.0 spec defines four roles:
+Initial Setup
+-------------
 
-- Resource owner
-- Resource server
-- Authorization server
-- Client
+Any new Partner application needs to be first set up with DOCK manually before any authentication or data exchange services can be used. Following are the one time set up steps for partners:
+Provide basic info like your application's name, logo and description.
+Provide these URLs: homepage, privacy policy and at least one callback URI. 
+The above information, except for the callback URIs, will be shown to the User at the Authorization Page. 
```

Comment:
I'd change to "Except for the callback URI, the above information will be shown to the User once they reach Dock's Authorization Page"